packages/jest-circus/src/utils.ts

```diff
@@ -43,7 +43,7 @@ export const makeDescribe = (
 };
 
 export const makeTest = (
-  fn: Circus.TestFn | undefined,
```

Comment:
is this correct? e.g. `test.todo` does not take a test function. I suppose it must be seeing as typescript is happy 🤷

Reply:
I'm actually not sure, I did remove the optional type from https://github.com/facebook/jest/blob/d077f2e6f36cc5717d61b973cda51f63bd1a280e/packages/jest-types/src/Circus.ts#L69 and https://github.com/facebook/jest/blob/d077f2e6f36cc5717d61b973cda51f63bd1a280e/packages/jest-types/src/Circus.ts#L234 so maybe that's why TS is ok with it, but could it be that `fn` is undefined at places?

Reply:
should be good, we add an empty function: https://github.com/facebook/jest/blob/d077f2e6f36cc5717d61b973cda51f63bd1a280e/packages/jest-circus/src/index.ts#L149